data/governments.txt

```diff
@@ -151,6 +151,20 @@ government "Korath"
 	
 	"player reputation" -10
 
+government "Korath (Berserk)"
+	swizzle 0
+	color .8 .5 .1
+	language "Korath"
+	
+	"attitude toward"
+		"Kor Sestor" -1
+		"Korath" 1
+		"Remnant" -1
+		"Merchant" -1
```

Comment:
I suggest making their attitude toward merchants -.01, since they shouldn't like you pirating the merchants (in fact, they will be angry if you plundered their targets first). Also, are there plans for these berserk ships to raid human space?

Reply:
Very tentatively. More along the lines of "it is a small possibility.

The actual point for this Berserk government was to enable the creation of Chaser fleets/squads that can be nailed down in a particular system. When I initially created the Clean-up bounty mission, we found that the Raider created by the old Bounty mission would pick up the Chasers spawned by the Clean-up mission, which led to some confusion. So this government is to ensure that they don't get picked up. It sorted of expanded from there to being a "We're not here to loot anymore - just kill everything" mentality. 

If there was such a "Permanently hostile to *everything* " variable, I'd use it for these ones.

Reply:
It sounds like there is some nuance about the relationship values here. Is -1 full hostile, 0 neutral, and -.01 some kind "we'll attack them but the player doing so will still make us angry" thing?

Reply:
You're right about -.01, anything lower than -.01 will increase your reputation if you attack their enemy, 0 is friendly but will not care if you attack the other (in most cases you don't need 0), and anything higher than .01 will make that government care if you attack the other.

Reply:
If you want them to be permanently hostile to the player, change all values to (-).01. They won't really care.